showpass/Showpass-E2E-tests
# TA-34-Verify that 'Personal Info' can be updated

**Description:**
End users should be able to update their First, Last Names as well as the phone number.
This user story will be verifying that all these fields can be updated and that the data gets saved and displayed.
Steps:
- Navigate to https://beta.showpass.com/account/profile/personal-info/
- Update First Name
- Update Last Name
- Update Phone Number
- Click ‘Save’
- Make sure the new values properly show up

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-34

**How to test:**
Run the following test suite and ensure the corresponding spec passes
cypress/e2e/user-settings/destructive-profile-updates.cy.js

**Test execution results:**

<img width="1020" alt="TA-34-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/31aeb562-88f5-4a62-b2fd-d81607bea0fc">
